roles/reproducer/tasks/default_repositories.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: "Ensure repository is present: {{ repository.src | basename }}"
   delegate_to: controller-0
+  remote_user: zuul
```

Comment:
so this will requrie that we create a zuul user on the image we user for the hosts and vms

that probably should be configurable but that could come later and it can default to zuul.

Reply:
controller-0 VM is 100% managed by the libvirt_manager, and the user is created there. No need to do anything on your side.

Reply:
(in fact I think this one isn't required - probably a leftover from when I hit so many issues with delegate_to, inventory and ssh_config)